app/javascript/mastodon/components/status_list.js

```diff
@@ -41,16 +42,26 @@ class StatusList extends ImmutablePureComponent {
     } else if (this.props.onScroll) {
       this.props.onScroll();
     }
-  }
+  }, 200, {
+    trailing: true,
+  });
```

Comment:
I can't see any reason not to add a debounce; we don't need precision beyond ~200 milliseconds to update the user's scroll position while they're scrolling. The `trailing` is to ensure that the scroll position truly gets updated even when the user stops scrolling.